src/tequila/circuit/visualize.py

```diff
@@ -0,0 +1,128 @@
+import importlib.util
+
+import tequila
+from typing import Optional, List
+import matplotlib.pyplot as plt
+import matplotlib.figure as figure
+import networkx
+
+
+def visualize(qubit_hamiltonian: tequila.QubitHamiltonian,
+              circuit: Optional[tequila.QCircuit] = None,
+              connectivity: Optional[List[tuple]] = None,
+              file_path: Optional[str] = None) -> figure.Figure:
```

Comment:
can you rename "file_path" to "filename" (consistency with other parts of the code where we export to files).
You can also allow: filename=None and file_path=None

and then assemble to filename=file_path/filename if file_path is not None

Reply:
Ok, I've changed the parameter name to filename and decided not to add file_path because I think it will make users confusing